app/assets/javascripts/dashboard/protocol_form.js.coffee

```diff
@@ -173,7 +173,7 @@ $(document).ready ->
 
   ###END PUBLISH IN EPIC BUTTON STATES###
 
-  if $("input[name='protocol[selected_for_epic]']").val() == 'true'
+  if $("input[name='protocol[selected_for_epic]'][value='true']").parent().hasClass('active')
```

Comment:
You know, I'm not sure the '>' is needed here... Might simplify to remove it. But I might be missing something.